accname/shadowdom/author1.html

```diff
@@ -0,0 +1,20 @@
+<!doctype html>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
+<script src="/resources/testdriver-actions.js"></script>
+
+<label id="label">
+  <div id="host"></div>
+</label>
+<button id="test" type="button" aria-labelledby="label"></button>
+<script>
+document.getElementById('host').attachShadow({ mode: 'open' }).innerHTML = '<div aria-label="foo"></div>';
+
+promise_test(async t => {
+  const label = await test_driver.get_computed_label(document.getElementById('test'));
+  assert_equals(label, "foo");
+}, "tests computed label - aria-label in shadow DOM");
```

Comment:
I should probably have mentioned this (sorry), but you can combine related tests in a single file. For example "slots.html" instead of slots1, slots2… That should make it easier to review completeness of the tests, too.

You can also avoid the extra `promise_test()` block if you use the aria-utils.js helpers like `verifyLabelsBySelector`

See this one as an example with two tests.
https://github.com/web-platform-tests/wpt/blob/master/wai-aria/role/region-roles.html

Reply:
Please to see all the new tests pass in [WebKit](https://wpt.fyi/results/accname/shadowdom?diff&filter=ADC&run_id=5102337574567936&run_id=5148191350063104) and [Chromium](https://wpt.fyi/results/accname/shadowdom?diff&filter=ADC&run_id=5155801361022976&run_id=5201736136916992). 🎉

Reply:
If you'd prefer to keep them as-is, I don't have a strong opinion though, unless you want to make the suggested changes, I will r+ and merge.

Reply:
I thought added my review as a non-blocking "comment." Perhaps it looks like "changes requested" because I did a line-specific comment. 🤷

Reply:
Actually please do combine the slot tests, but I don't care if you use the ARIAUtils or keep the local `promise_test()` block. Thanks for working on these.

Reply:
No problem. I added a `verifyLabelsBySelector` helper to `AriaUtils` and combined the tests into `basic.html` and `slot.html` (d01da6b). Let me know if this looks cleaner. And thanks for the review!

Reply:
Oh shoot. Sorry. I forgot verifyLabelsBySelector hadn't landed yet, so there will be a conflict with #39604. Wait until that one lands and then delete this branch's changes to are-util.js and it should work.

Reply:
FYI @nolanlawson the initial AccName structure is now merged, along with `verifyLabelsBySelector` in #39604. Since this is testing name computation, can you move the new `shadowdom` dir into `acne/name`? I  added that so we can have similar directories once we're able to test description computation (in `accname/description`) and value computation (in `accname/value`)

Reply:
Sounds good! Updated this PR.